src/lib.rs

```diff
@@ -540,3 +542,42 @@ impl Default for WindowAttributes {
         }
     }
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct VideoMode {
+    dimensions: (u32, u32),
```

Comment:
Can this become `PhysicalSize` instead of `(u32, u32)`

Reply:
This needs to implement `Hash` which `PhysicalSize` does not (and can not, as it's essentially a tuple of `f64`). Hence, we're doing the conversion from `(u32, u32)` to `PhysicalSize` in the getter method.

Reply:
`PhysicalSize` is stored as `u32`s and not `f64`s on `eventloop-2.0`, so this should be possible once rebased.

Reply:
Looks like this would depend on the changes made in #895 being merged first.